attack-theme/templates/general/base-template.html

```diff
@@ -108,7 +111,7 @@
                         </div>
                     {% else %}
                         <div class="">
-                            <a href="attack.mitre.org" class="btn btn-primary w-100">
+                            <a href="https://attack.mitre.org" class="btn btn-primary w-100">
                                 attack.mitre.org
                             </a>
```

Comment:
Button needs vertical centering like the MITRE logo. See [how to center in CSS](http://howtocenterincss.com/#contentType=div&horizontal=center&vertical=middle&browser.IE=6) for vertical centering tricks.